cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/ResiliencyTestBase.java

```diff
@@ -375,10 +375,22 @@ protected void bulkWriteData(ConsistencyLevel writeCL, QualifiedName schema)
             if (!completed)
             {
                 process.destroyForcibly();
-                finishLatch.await(10, TimeUnit.MINUTES);
+                // The process doesn't necessarily finish right after destroyForcibly, but will exit.
+                process.waitFor(1, TimeUnit.MINUTES);
```

Comment:
maybe add a comment on why we only let the spawned spark process run for 1 minute. And the `errorMessage` in line 380 will need to be updated as well

Reply:
The comment on 380 is still valid - after 10 minutes, we attempt to kill the process. I did add "and was killed" along with a bit more commenting on the additional 1 minute of wait.